todo_app/app.py

```diff
@@ -1,4 +1,6 @@
-from flask import Flask
+from flask import Flask, redirect, request, url_for
+from flask import render_template
```

Comment:
More down to preference rather than an actual recommendation, but worth noting we can combine these still:
```suggestion
from flask import Flask, redirect, request, url_for, render_template
```